### PR TITLE
Allow to filter get-updates on device-id

### PR DESCRIPTION
**Pull request by djcampello**
This change allows to get available updates for a given device. This is
similar to what is done for the update and activate commands.

Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ X ] Feature
- [ ] Documentation

### CODE REVIEW

**Review by hughsie — APPROVED** [review]

**Comment by bleungatchromium** [comment]
`Change-Id: I6743d4c34b5fa1ecff419f8a1c7bb2231d6f279c
`

Go ahead and strip gerrit's Change-Id from the submission here.

File: src/fu-util.c [hunk 1]
```diff
@@ -1376,10 +1376,28 @@ fu_util_get_updates (FuUtilPrivate *priv, gchar **values, GError **error)
 	if (!fu_util_perhaps_refresh_remotes (priv, error))
 		return FALSE;
 
-	/* get devices from daemon */
-	devices = fwupd_client_get_devices (priv->client, NULL, error);
-	if (devices == NULL)
+	/* handle both forms */
+	if (g_strv_length (values) == 0) {
+		/* activate anything with _NEEDS_ACTIVATION */
```
Comment (superm1) [inline src/fu-util.c:1381]: this comment seems wrong

**Review by superm1 — COMMENTED** [review]
conceptually seems like a reasonable enhancement.  However can you please modify the bash-completion files to make sure that it's updated for this new scenario?

File: src/fu-util.c (hunk 1 above)
Comment (djcampello) [reply src/fu-util.c]: Sorry, forgot to read after copy/paste :( Thanks for catching

**Comment by djcampello** [comment]
> `Change-Id: I6743d4c34b5fa1ecff419f8a1c7bb2231d6f279c `
> 
> Go ahead and strip gerrit's Change-Id from the submission here.

How do you do this on the commit itself?

**Comment by superm1** [comment]
>How do you do this on the commit itself?

`git commit --amend`

But If it's technically difficult to do, it's okay to leave in.

File: data/bash-completion/fwupdtool.in [hunk 2]
```diff
@@ -109,7 +109,7 @@ _fwupdtool()
 			_show_modifiers
 		fi
 		;;
-	attach|detach|activate|verify-update|reinstall)
+	attach|detach|activate|verify-update|reinstall|get-updates)
```
Comment (superm1) [inline data/bash-completion/fwupdtool.in:112]: can you also update the fwupdmgr bash completion file for the same change?

**Comment by djcampello** [comment]
> > `Change-Id: I6743d4c34b5fa1ecff419f8a1c7bb2231d6f279c `
> > Go ahead and strip gerrit's Change-Id from the submission here.
> 
> How do you do this on the commit itself?

I just managed with: 
git -c gerrit.createChangeId=FALSE commit --amend

File: data/bash-completion/fwupdtool.in (hunk 2 above)
Comment (djcampello) [reply data/bash-completion/fwupdtool.in:112]: Done

**Review by superm1 — APPROVED** [review]

**Pull request merged**